index.html

```diff
@@ -0,0 +1,179 @@
+            </div>
+            <ul class="work-nav">
+                <li class="link-border"><a href="#">css</a></li>
+                <li class="link-border"><a href="#">html</a></li>
+                <li class="link-border"><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+            <button type="button" class="see-project-button">See Project</button>
+        </div>
+        <br><br><br>
+        <div class="recent-work-placeholder">
+            <div>
+                <img src="images/ImgPlaceholder.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="work-nav">
+                <li class="link-border"><a href="#">css</a></li>
+                <li class="link-border"><a href="#">html</a></li>
+                <li class="link-border"><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+            <button type="button" class="see-project-button">See Project</button>
+        </div>
+        <br><br><br>
+        <div class="recent-work-placeholder">
+            <div>
+                <img src="images/ImgPlaceholder.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="work-nav">
+                <li class="link-border"><a href="#">css</a></li>
+                <li class="link-border"><a href="#">html</a></li>
+                <li class="link-border"><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+            <button type="button" class="see-project-button">See Project</button>
+        </div>
+        <br><br><br>
+        <div class="recent-work-placeholder">
+            <div>
+                <img src="images/ImgPlaceholder.png" alt="">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            </div>
+            <ul class="work-nav">
+                <li class="link-border"><a href="#">css</a></li>
+                <li class="link-border"><a href="#">html</a></li>
+                <li class="link-border"><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+            <button type="button" class="see-project-button">See Project</button>
```

Comment:
- [ ] Kindly follow the button styling for all buttons: Active, Hover and Pressed as per the Figma design.
<img width="699" alt="Screenshot 2022-03-27 at 1 14 59 AM" src="https://user-images.githubusercontent.com/70269507/160258757-48e8f97a-f31d-4c83-9978-b3d9a865b6b7.png">